server/graphql/v2/object/PersonalToken.ts

```diff
@@ -0,0 +1,50 @@
+import { GraphQLList, GraphQLNonNull, GraphQLObjectType, GraphQLString } from 'graphql';
+
+import { DateString } from '../../v1/types';
```

Comment:
:no_entry_sign: Never this one on GQLV2. We can instead use:

```suggestion
import { GraphQLDateTime } from 'graphql-scalars';
```